docs/CSS.md

```diff
@@ -0,0 +1,98 @@
+### 1. `flex`
+* *elements:* none
+* *inherited:* no 
+
+#### Values
+
+* `<flex-grow>` -- defines the flex-grow of the flex item
+* `<flex-shrink>` -- defines the flex-shrink of the flex item
+* `<flex-basis>` -- defines the flex-basis of the flex item
+* `none` -- this keyword computes to 0 0 auto;
+
```

Comment:
Man. There's a doozy. This is one of a handful of _composite properties_ that provide convenient shortcuts for a family of related properties. In this case, the `flex-` properties like `flex-grow` etc can be succinctly written with just `flex`... but you should start with the expanded properties that roll up into `flex`. It makes it much easier to experiment with them without accidentally breaking the syntax when they're separate.